.eslintignore

```diff
@@ -4,4 +4,6 @@
 /examples
 /mithril.js
 /mithril.min.js
+/stream.js
```

Comment:
When you `require("mithril/stream")`, it searches for the following files ([source](https://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders)):

- `node_modules/mithril/stream.js`
- `node_modules/mithril/stream.node` (and other registered extensions)
- `node_modules/mithril/stream/{pkg.main}` (where `"main"` is from the `package.json`)
- `node_modules/mithril/stream/index.js`
- `node_modules/mithril/stream/index.node` (and other registered extensions)

This means when you save a file as `stream.js` in the root, when it gets published, Node will find `mithril/stream.js` rather than the intended `mithril/stream/index.js`.